src/item/Bow.php

```diff
@@ -36,6 +37,10 @@
 
 class Bow extends Tool implements Releasable{
 
+	public function __construct(ItemIdentifier $identifier, string $name = "Unknown"){
+		parent::__construct($identifier, $name, ItemFlags::BOW);
```

Comment:
Baking information into class constructors is generally discouraged.

Reply:
That was suggested by @ShockedPlot7560 )